code/Util/PlayerExtensions.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using Sandbox;
+
+namespace TTT;
+
+public static class PlayerExtensions
+{
+	public static bool KilledWithHeadShot( this Player player )
```

Comment:
This could be a property on the player, the same way as `KilledByPlayer`.

Reply:
sure